src/igvfd/ontology.py

```diff
@@ -0,0 +1,33 @@
+import gzip
+import json
+
+from pathlib import Path
+
+
+def includeme(config):
+    config.scan(__name__)
+    config.registry['ontology'] = load_ontology()
+
+
+ONTOLOGY_NAME = 'ontology.json.gz'
```

Comment:
nit, but I suppose `ONTLOGY_FILE_NAME` would be more appropriate.

Reply:
Updated.